setup.py

```diff
@@ -93,6 +93,7 @@ def read(*names, **kwargs):
 
         # packagedcode
         'requests >= 2.7.0, < 3.0.0',
+        'schematics',
```

Comment:
You need to put a version range constraint, we do not accept any schematics version but the current up to the next major excluded